kernelci/configs.py

```diff
@@ -496,6 +496,7 @@ def __init__(self, name, mach, arch, boot_method, dtb=None,
         self._arch = arch
         self._boot_method = boot_method
         self._dtb = dtb
+        self._params = params
```

Comment:
Please follow the same rule as with other classes and initialise self._params as an empty dict by default.

Reply:
done